test/framework/convenience.go

```diff
@@ -73,3 +78,27 @@ func EnsureNamespace(ctx context.Context, mgmt client.Client, namespace string)
 		Fail(err.Error())
 	}
 }
+
+// TryAddDefaultSchemes tries to add the following schemes:
+//   * Kubernetes corev1
+//   * Kubernetes appsv1
+//   * CAPI core
+//   * Kubeadm Bootstrapper
+//   * Kubeadm ControlPlane
+// Any error that occurs when trying to add the schemes is ignored.
+func TryAddDefaultSchemes(scheme *runtime.Scheme) {
+	// Add the core schemes.
+	corev1.AddToScheme(scheme)
```

Comment:
Is this expected to be run under gomega? If so, would it make sense to go ahead and add expectations that these succeed? If we are expecting these types to be available and they aren't it'd be nice to know sooner rather than trying to debug cryptic failures later.

Reply:
Hi @detiber,

There's no expectation that this runs under `gomega`. Similarly that's why `LoadConfig` doesn't have any assertions. Additionally, the function name is `Try...` specifically to indicate that any errors are ignored. It's built like this so the function can be invoked multiple times, even if the schemes have already been added.